great_expectations/expectations/expectation.py

```diff
@@ -3164,6 +3172,7 @@ def _format_map_output(
     unexpected_list: Optional[List[Any]] = None,
     unexpected_index_list: Optional[List[int]] = None,
     unexpected_index_query: Optional[str] = None,
+    unexpected_index_column_names: Optional[Union[int, str, List[str]]] = None,
```

Comment:
This is not entirely accurate, but is necessary to keep the typechecker happy (it's actually `Optional[List[str]]`